Explorer/Assets/DCL/SDKComponents/Tween/Components/CustomTweener/CustomTweenerImpl.cs

```diff
@@ -41,31 +38,28 @@ protected override TweenerCore<Vector3, Vector3, VectorOptions> CreateTweener(Ve
             return DOTween.To(() => currentValue, x => currentValue = x, end, duration);
         }
 
-        protected override (Vector3, Vector3) GetTweenValues(PBTween pbTween, Transform startTransform)
+        protected override (Vector3, Vector3) GetTweenValues(PBTween pbTween, SDKTransform startTransform)
         {
             Vector3 start = PrimitivesConversionExtensions.PBVectorToUnityVector(pbTween.Scale.Start);
             Vector3 end = PrimitivesConversionExtensions.PBVectorToUnityVector(pbTween.Scale.End);
-            startTransform.localScale = start;
+            startTransform.Scale = start;
             currentValue = start;
             return (start, end);
         }
 
         public override void SetResult(ref SDKTransform sdkTransform)
         {
             sdkTransform.Scale = currentValue;
-
```

Comment:
please verify that droid scene still works fine
[#1476 goerli: the droid doesn't always flies away from the user](https://github.com/decentraland/unity-explorer/issues/1476)

Reply:
The droid scene seems to be broken again. The droid does not seem to fly away more than once 😢